proteus/BoundaryConditions.pyx

```diff
@@ -47,7 +47,7 @@ class BC_Base(object):
             cls.ct = context
 
 
-class BoundaryCondition():
+class BoundaryCondition(BC_Base):
```

Comment:
Thanks for looking at it. Based on our phone call I'll wait for you to double check that I haven't made  any other mistakes with BoundaryConditions/SpatialTools and put them back as they should be. You can go ahead and merge once any of your modifications pass the  tests.